layers/+lang/latex/packages.el

```diff
@@ -144,13 +145,20 @@
     :defer t
     :init (spacemacs|add-company-backends
             :backends
-            company-auctex-labels
```

Comment:
Why remove these lines?

Reply:
They are replaced by their company-reftex counterparts:

`company-reftex-labels` and `company-reftex-citations`